### PR TITLE
Refactor `composer.json`

### PR DESCRIPTION
**Pull request by drupol**
This PR:

- [x] Normalize `composer.json`
- [x] Fix indentation (based on `.editorconfig`)

### CODE REVIEW

**Review by marcosh — COMMENTED** [review]
It's not a must, but I'd like to keep the `composer.lock`. It should not be an issue since I'm testing all the available extremes of the dependencies

File: composer.json [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+    "readme": "README.md",
+    "authors": [
+        {
+            "name": "Marco Perone",
+            "email": "pasafama@gmail.com"
+        }
+    ],
+    "homepage": "https://github.com/marcosh/lamphpda",
+    "require": {
+        "php": ">= 7.4",
```
[the remark this answers is not in the document]
Comment (drupol) [reply composer.json]: Ok sad. This is something that was done in Symfony already: https://github.com/symfony/symfony/pull/36876

**Comment by drupol** [comment]
Hi ! 

I reverted the changes based on your feedback.

**Comment by drupol** [comment]
Thanks!

**Comment by marcosh** [comment]
@drupol thank you!

**Pull request merged**